lib/features/thread/presentation/thread_controller.dart

```diff
@@ -333,63 +324,12 @@ class ThreadController extends BaseController with EmailActionController {
 
     ever(mailboxDashBoardController.emailUIAction, (action) {
       if (action is RefreshChangeEmailAction) {
-        if (action.newState != _currentEmailState) {
-          _refreshEmailChanges();
-        }
-        mailboxDashBoardController.clearEmailUIAction();
```

Comment:
IMO, we still need to `clear` action?

Reply:
No, Its action contains a `newState` and is an EqualMixin. However, we need to listen for it in multiple controllers so we can't clear it here.